.github/workflows/mediasoup-rust.yaml

```diff
@@ -23,6 +23,9 @@ jobs:
 
     runs-on: ${{ matrix.os }}
 
+    env:
+      KEEP_BUILD_ARTIFACTS: '1'
```

Comment:
Why was this necessary?

Reply:
It was there already: https://github.com/versatica/mediasoup/pull/1216/files#diff-03b3d6e48484149f7e40c987fc6b29683f6a22bb39bd9095e210ae3b1cef70b3L53

Yes, just in `cargo test` step, but IMHO it makes sense that we set that env from scratch because we don'w want to fetch and rebuild everything if already built in a previous step.

Reply:
I see, missed that the first time :see_no_evil: